loop_live.py

```
@@ -0,0 +1,644 @@
+
+
+    if int(choice) == len(prompt_files) + 1:
+        prompt_text = Prompt.ask("Enter your prompt")
+        new_prompt_path = prompts_dir / f"{filename}.md"
+        with open(new_prompt_path, 'w', encoding='utf-8') as f:
+            f.write(prompt_text)
+        task = prompt_text
+    else:
+        with open(prompt_path, 'r', encoding='utf-8') as f:
+            task = f.read()
+    project_dir = set_project_dir(filename)
```

Ruff (0.8.2)

`set_project_dir` may be undefined, or defined from star imports

(F405)